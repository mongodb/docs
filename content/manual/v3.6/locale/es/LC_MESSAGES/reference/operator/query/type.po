# SOME DESCRIPTIVE TITLE.
# Copyright (C) 2011-2014, MongoDB, Inc.
# This file is distributed under the same license as the mongodb-manual
# package.
#
# Translators:
msgid ""
msgstr ""
"Project-Id-Version: MongoDB Manual\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2019-03-19 11:30-0400\n"
"PO-Revision-Date: 2014-04-08 18:55+0000\n"
"Last-Translator: tychoish <tychoish@gmail.com>\n"
"Language: es\n"
"Language-Team: Spanish (http://www.transifex.com/projects/p/mongodb-"
"manual/language/es/)\n"
"Plural-Forms: nplurals=2; plural=(n != 1)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.6.0\n"

#: ../source/reference/operator/query/type.txt:3
msgid "$type"
msgstr ""

#: ../source/reference/operator/query/type.txt:14
msgid "Definition"
msgstr ""

#: ../source/reference/operator/query/type.txt:18
msgid ""
":query:`$type` selects the documents where the *value* of the ``field`` "
"is an instance of the specified :term:`BSON` type(s). Querying by data "
"type is useful when dealing with highly unstructured data where data "
"types are not predictable."
msgstr ""

#: ../source/reference/operator/query/type.txt:23
msgid ""
"A :query:`$type` expression for a single :term:`BSON` type has the "
"following syntax:"
msgstr ""

#: ../source/reference/operator/query/type.txt:32
msgid ""
"You can specify either the number or alias for the :ref:`BSON type"
"<document-type-available-types>`"
msgstr ""

#: ../source/reference/operator/query/type.txt:35
msgid ""
"The :query:`$type` expression can also accept an array of :term:`BSON` "
"types and has the following syntax:"
msgstr ""

#: ../source/reference/operator/query/type.txt:42
msgid ""
"The above query will match documents where the ``field`` value is any of "
"the listed types. The types specified in the array can be either numeric "
"or string aliases."
msgstr ""

#: ../source/reference/operator/query/type.txt:46
msgid "See :ref:`document-querying-by-multiple-data-types` for an example."
msgstr ""

#: ../source/reference/operator/query/type.txt:48
msgid ""
":ref:`document-type-available-types` describes the BSON types and their "
"corresponding numeric and string aliases."
msgstr ""

#: ../source/reference/operator/query/type.txt:53
msgid ""
"If you wish to obtain the :ref:`BSON type <bson-types>` returned by an "
":ref:`operator expression <agg-quick-ref-operator-expressions>` rather "
"than filtering documents by their BSON type, use the :expression:`$type` "
"aggregation operator."
msgstr ""

#: ../source/reference/operator/query/type.txt:59
msgid "Behavior"
msgstr ""

#: ../source/reference/operator/query/type.txt:61
msgid ""
":query:`$type` returns documents where the BSON type of the ``field`` "
"matches the BSON type passed to :query:`$type`."
msgstr ""

#: ../source/reference/operator/query/type.txt:66
msgid ""
":query:`$type` now works with arrays in the same way it works with other "
"BSON types. Previous versions only matched documents where the ``field`` "
"contained a nested array."
msgstr ""

#: ../source/reference/operator/query/type.txt:73
msgid "Available Types"
msgstr ""

#: ../source/reference/operator/query/type.txt:75
msgid ""
":query:`$type` operator accepts string aliases for the BSON types in "
"addition to the numbers corresponding to the BSON types. Previous "
"versions only accepted the numbers corresponding to the BSON type."
msgstr ""

#: ../source/includes/fact-bson-types.rst:4
msgid "Type"
msgstr ""

#: ../source/includes/fact-bson-types.rst:5
msgid "Number"
msgstr ""

#: ../source/includes/fact-bson-types.rst:6
msgid "Alias"
msgstr ""

#: ../source/includes/fact-bson-types.rst:7
msgid "Notes"
msgstr ""

#: ../source/includes/fact-bson-types.rst:9
msgid "Double"
msgstr ""

#: ../source/includes/fact-bson-types.rst:10
msgid "1"
msgstr ""

#: ../source/includes/fact-bson-types.rst:11
msgid "\"double\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:14
msgid "String"
msgstr ""

#: ../source/includes/fact-bson-types.rst:15
msgid "2"
msgstr ""

#: ../source/includes/fact-bson-types.rst:16
msgid "\"string\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:19
msgid "Object"
msgstr ""

#: ../source/includes/fact-bson-types.rst:20
msgid "3"
msgstr ""

#: ../source/includes/fact-bson-types.rst:21
msgid "\"object\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:24
msgid "Array"
msgstr ""

#: ../source/includes/fact-bson-types.rst:25
msgid "4"
msgstr ""

#: ../source/includes/fact-bson-types.rst:26
msgid "\"array\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:29
msgid "Binary data"
msgstr ""

#: ../source/includes/fact-bson-types.rst:30
msgid "5"
msgstr ""

#: ../source/includes/fact-bson-types.rst:31
msgid "\"binData\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:34
msgid "Undefined"
msgstr ""

#: ../source/includes/fact-bson-types.rst:35
msgid "6"
msgstr ""

#: ../source/includes/fact-bson-types.rst:36
msgid "\"undefined\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:37
#: ../source/includes/fact-bson-types.rst:67
#: ../source/includes/fact-bson-types.rst:77
msgid "Deprecated."
msgstr ""

#: ../source/includes/fact-bson-types.rst:39
msgid "ObjectId"
msgstr ""

#: ../source/includes/fact-bson-types.rst:40
msgid "7"
msgstr ""

#: ../source/includes/fact-bson-types.rst:41
msgid "\"objectId\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:44
msgid "Boolean"
msgstr ""

#: ../source/includes/fact-bson-types.rst:45
msgid "8"
msgstr ""

#: ../source/includes/fact-bson-types.rst:46
msgid "\"bool\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:49
msgid "Date"
msgstr ""

#: ../source/includes/fact-bson-types.rst:50
msgid "9"
msgstr ""

#: ../source/includes/fact-bson-types.rst:51
msgid "\"date\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:54
msgid "Null"
msgstr ""

#: ../source/includes/fact-bson-types.rst:55
msgid "10"
msgstr ""

#: ../source/includes/fact-bson-types.rst:56
msgid "\"null\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:59
msgid "Regular Expression"
msgstr ""

#: ../source/includes/fact-bson-types.rst:60
msgid "11"
msgstr ""

#: ../source/includes/fact-bson-types.rst:61
msgid "\"regex\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:64
msgid "DBPointer"
msgstr ""

#: ../source/includes/fact-bson-types.rst:65
msgid "12"
msgstr ""

#: ../source/includes/fact-bson-types.rst:66
msgid "\"dbPointer\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:69
msgid "JavaScript"
msgstr ""

#: ../source/includes/fact-bson-types.rst:70
msgid "13"
msgstr ""

#: ../source/includes/fact-bson-types.rst:71
msgid "\"javascript\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:74
msgid "Symbol"
msgstr ""

#: ../source/includes/fact-bson-types.rst:75
msgid "14"
msgstr ""

#: ../source/includes/fact-bson-types.rst:76
msgid "\"symbol\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:79
msgid "JavaScript (with scope)"
msgstr ""

#: ../source/includes/fact-bson-types.rst:80
msgid "15"
msgstr ""

#: ../source/includes/fact-bson-types.rst:81
msgid "\"javascriptWithScope\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:84
#: ../source/reference/operator/query/type.txt:88
msgid "32-bit integer"
msgstr ""

#: ../source/includes/fact-bson-types.rst:85
msgid "16"
msgstr ""

#: ../source/includes/fact-bson-types.rst:86
msgid "\"int\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:89
msgid "Timestamp"
msgstr ""

#: ../source/includes/fact-bson-types.rst:90
msgid "17"
msgstr ""

#: ../source/includes/fact-bson-types.rst:91
msgid "\"timestamp\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:94
#: ../source/reference/operator/query/type.txt:90
msgid "64-bit integer"
msgstr ""

#: ../source/includes/fact-bson-types.rst:95
msgid "18"
msgstr ""

#: ../source/includes/fact-bson-types.rst:96
msgid "\"long\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:99
msgid "Decimal128"
msgstr ""

#: ../source/includes/fact-bson-types.rst:100
msgid "19"
msgstr ""

#: ../source/includes/fact-bson-types.rst:101
msgid "\"decimal\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:102
msgid "New in version 3.4."
msgstr ""

#: ../source/includes/fact-bson-types.rst:104
msgid "Min key"
msgstr ""

#: ../source/includes/fact-bson-types.rst:105
msgid "-1"
msgstr ""

#: ../source/includes/fact-bson-types.rst:106
msgid "\"minKey\""
msgstr ""

#: ../source/includes/fact-bson-types.rst:109
msgid "Max key"
msgstr ""

#: ../source/includes/fact-bson-types.rst:110
msgid "127"
msgstr ""

#: ../source/includes/fact-bson-types.rst:111
msgid "\"maxKey\""
msgstr ""

#: ../source/reference/operator/query/type.txt:83
msgid ""
":query:`$type` supports the ``number`` alias, which will match against "
"the following :term:`BSON` types:"
msgstr ""

#: ../source/reference/operator/query/type.txt:86
msgid "double"
msgstr ""

#: ../source/reference/operator/query/type.txt:92
msgid "decimal"
msgstr ""

#: ../source/reference/operator/query/type.txt:94
msgid "See :ref:`document-querying-by-data-type`"
msgstr ""

#: ../source/reference/operator/query/type.txt:97
msgid "MinKey and MaxKey"
msgstr ""

#: ../source/reference/operator/query/type.txt:99
msgid ""
":bsontype:`MinKey <data_minkey>` and :bsontype:`MaxKey <data_maxkey>` are"
" used in comparison operations and exist primarily for internal use. For "
"all possible :term:`BSON` element values, ``MinKey`` will always be the "
"smallest value while ``MaxKey`` will always be the greatest value."
msgstr ""

#: ../source/reference/operator/query/type.txt:104
msgid ""
"Querying for  ``minKey`` or ``maxKey`` with :query:`$type` will only "
"return fields that match the special ``MinKey`` or ``MaxKey`` values."
msgstr ""

#: ../source/reference/operator/query/type.txt:108
msgid ""
"Suppose that the ``data`` collection has two documents with ``MinKey`` "
"and ``MaxKey``:"
msgstr ""

#: ../source/reference/operator/query/type.txt:116
msgid "The following query will return the document with ``_id: 1``:"
msgstr ""

#: ../source/reference/operator/query/type.txt:122
msgid "The following query will return the document with ``_id: 2``:"
msgstr ""

#: ../source/reference/operator/query/type.txt:129
msgid "Examples"
msgstr ""

#: ../source/reference/operator/query/type.txt:134
msgid "Querying by Data Type"
msgstr ""

#: ../source/reference/operator/query/type.txt:136
msgid ""
"The ``addressBook`` contains addresses and zipcodes, where ``zipCode`` "
"has ``string``, ``int``, ``double``, and ``long`` values:"
msgstr ""

#: ../source/reference/operator/query/type.txt:151
msgid ""
"The following queries return all documents where ``zipCode`` is the "
":term:`BSON` type ``string``:"
msgstr ""

#: ../source/reference/operator/query/type.txt:160
#: ../source/reference/operator/query/type.txt:174
#: ../source/reference/operator/query/type.txt:187
msgid "These queries return:"
msgstr ""

#: ../source/reference/operator/query/type.txt:166
msgid ""
"The following queries return all documents where ``zipCode`` is the "
":term:`BSON` type ``double``:"
msgstr ""

#: ../source/reference/operator/query/type.txt:180
msgid ""
"The following query uses the ``number`` alias to return documents where "
"``zipCode`` is the :term:`BSON` type ``double``, ``int``, or ``long``:"
msgstr ""

#: ../source/reference/operator/query/type.txt:198
msgid "Querying by Multiple Data Type"
msgstr ""

#: ../source/reference/operator/query/type.txt:200
msgid ""
"The ``grades`` collection contains names and averages, where "
"``classAverage`` has ``string``, ``int``, and ``double`` values:"
msgstr ""

#: ../source/reference/operator/query/type.txt:214
msgid ""
"The following queries return all documents where ``classAverage`` is the "
":term:`BSON` type ``string`` or ``double``. The first query uses numeric "
"aliases while the second query uses string aliases."
msgstr ""

#: ../source/reference/operator/query/type.txt:223
msgid "These queries return the following documents:"
msgstr ""

#: ../source/reference/operator/query/type.txt:234
msgid "Querying by MinKey and MaxKey"
msgstr ""

#: ../source/reference/operator/query/type.txt:236
msgid ""
"The ``restaurants`` collection uses ``minKey`` for any grade that is a "
"failing grade:"
msgstr ""

#: ../source/reference/operator/query/type.txt:261
msgid "And ``maxKey`` for any grade that is the highest passing grade:"
msgstr ""

#: ../source/reference/operator/query/type.txt:285
msgid ""
"The following query returns any restaurant whose ``grades.grade`` field "
"contains ``minKey``:"
msgstr ""

#: ../source/reference/operator/query/type.txt:294
#: ../source/reference/operator/query/type.txt:328
msgid "This returns"
msgstr ""

#: ../source/reference/operator/query/type.txt:318
msgid ""
"The following query returns any restaurant whose ``grades.grade`` field "
"contains ``maxKey``:"
msgstr ""

#: ../source/reference/operator/query/type.txt:355
msgid "Querying by Array Type"
msgstr ""

#: ../source/reference/operator/query/type.txt:357
msgid "A collection named ``SensorReading`` contains the following documents:"
msgstr ""

#: ../source/reference/operator/query/type.txt:396
msgid ""
"The following query returns any document in which the ``readings`` field "
"is an array, empty or non-empty."
msgstr ""

#: ../source/reference/operator/query/type.txt:403
msgid "The above query returns the following documents:"
msgstr ""

#: ../source/reference/operator/query/type.txt:438
msgid ""
"In the documents with ``_id : 1``, ``_id : 2``, ``_id : 3``, and ``_id : "
"4``, the ``readings`` field is an array."
msgstr ""

#: ../source/reference/operator/query/type.txt:442
msgid "Additional Information"
msgstr ""

#: ../source/reference/operator/query/type.txt:444
msgid ""
":method:`~db.collection.find()`, :doc:`BSON Types </reference/bson-"
"types>`."
msgstr ""

# 23e9e551a2ad441ca9e1a50d260c74ec
#~ msgid "*Syntax*: ``{ field: { $type: <BSON type> } }``"
#~ msgstr ""

# fa3d55241c7048cdaaad0fc6609d1399
#~ msgid ""
#~ ":query:`$type` selects the documents where "
#~ "the *value* of the ``field`` is "
#~ "the specified :term:`BSON` type."
#~ msgstr ""

# 52569ba39ff84ad18b57ee178f5007b9
#~ msgid "Consider the following example:"
#~ msgstr ""

# 4e4c63aa9e7748578f331916f45a5446
#~ msgid ""
#~ "This query will select all documents "
#~ "in the ``inventory`` collection where "
#~ "the ``price`` field value is a "
#~ "Double."
#~ msgstr ""

# ff45173dff5946928563e5cce4f346bb
#~ msgid ""
#~ "If the ``field`` holds an array, "
#~ "the :query:`$type` operator performs the "
#~ "type check against the array elements"
#~ " and **not** the ``field``."
#~ msgstr ""

# 18f3fe2242c04653a3c197b9a4e8cfc9
#~ msgid "Consider the following example where the ``tags`` field holds an array:"
#~ msgstr ""

# 69692a695c5445968ecdd0f928a605bc
#~ msgid ""
#~ "This query will select all documents "
#~ "in the ``inventory`` collection where "
#~ "the ``tags`` array contains an element"
#~ " that is itself an array."
#~ msgstr ""

# f4f40b49d0e14394aa5c00f3cf5779ac
#~ msgid ""
#~ "If instead you want to determine "
#~ "whether the ``tags`` field is an "
#~ "array type, use the :query:`$where` "
#~ "operator:"
#~ msgstr ""

# d4bb9a3a13814faea51c81874d9c599d
#~ msgid "See the :issue:`SERVER-1475` for more information about the array type."
#~ msgstr ""

# c062fd7f91c345f8a86711a5cd243b67
#~ msgid ""
#~ "Refer to the following table for "
#~ "the available :term:`BSON` types and "
#~ "their corresponding numbers."
#~ msgstr ""

# 6ba1a306440a4f7e9d2d2b81e121da6d
#~ msgid "**Type**"
#~ msgstr ""

# b3ae2e06af2541939888815932afa5e6
#~ msgid "**Number**"
#~ msgstr ""

# 897f67d9539646118c5dfafa5792684f
#~ msgid "Undefined (deprecated)"
#~ msgstr ""

# ce6fcf68778442aa9f9d2a421a8d6e3e
#~ msgid "Object id"
#~ msgstr ""

# 676c2639fd2c4ae69348b0efa9cd9ab6
#~ msgid "255"
#~ msgstr ""

# 08c41e855d9448108a6d320e819489e1
#~ msgid ""
#~ "``MinKey`` and ``MaxKey`` compare less "
#~ "than and greater than all other "
#~ "possible :term:`BSON` element values, "
#~ "respectively, and exist primarily for "
#~ "internal use."
#~ msgstr ""

# de71905419aa4af99b798d143f24ef2b
#~ msgid ""
#~ "To query if a field value is "
#~ "a ``MinKey``, you must use the "
#~ ":query:`$type` with ``-1`` as in the "
#~ "following example:"
#~ msgstr ""

# 6083273e1aec4e8096988088dbc552a6
#~ msgid "Example"
#~ msgstr ""

# 52562d3d349d49e88be5b501287ab5ea
#~ msgid ""
#~ "Consider the following example operation "
#~ "sequence that demonstrates both type "
#~ "comparison *and* the special ``MinKey`` "
#~ "and ``MaxKey`` values:"
#~ msgstr ""

# 63b729f5015e45a7a07a889e334bc950
#~ msgid ""
#~ "To query for the minimum value of"
#~ " a :term:`shard key` of a "
#~ ":term:`sharded cluster`, use the following "
#~ "operation when connected to the "
#~ ":program:`mongos`:"
#~ msgstr ""

# 6811d8ab85d64507943ac590ef13e00d
#~ msgid ""
#~ "Storing values of the different types"
#~ " in the same field in a "
#~ "collection is *strongly* discouraged."
#~ msgstr ""

# 2c1e59c3232e4d0585ebb4b075413994
#~ msgid ""
#~ ":method:`~db.collection.find()`, "
#~ ":method:`~db.collection.insert()`, :query:`$where`, "
#~ ":term:`BSON`, :term:`shard key`, :term:`sharded "
#~ "cluster` ."
#~ msgstr ""

