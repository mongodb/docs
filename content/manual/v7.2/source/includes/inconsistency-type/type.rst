
Type of inconsistency document.  Returns a value of |incon-type| for this   
inconsistency type.


