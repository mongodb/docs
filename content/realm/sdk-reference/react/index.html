<!doctype html>
<head>
  <meta http-equiv="refresh" content="0; url=https://mongodb.com/docs/realm-sdks/react/latest" />
</head>
<body>
  <p><a href="https://mongodb.com/docs/realm-sdks/react/latest">Redirect</a></p>
</body>
