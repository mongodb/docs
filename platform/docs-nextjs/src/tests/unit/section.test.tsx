import { render } from '@testing-library/react';
import Section from '@/components/section';
import type { ParentNode } from '@/types/ast';

import mockData from '../data/section.test.json';
const typedMockData = mockData as ParentNode;

// TODO: Remove .skip once includes are supported
it.skip('renders correctly', () => {
  const tree = render(<Section nodeChildren={typedMockData.children} />);
  expect(tree.asFragment()).toMatchSnapshot();
});
