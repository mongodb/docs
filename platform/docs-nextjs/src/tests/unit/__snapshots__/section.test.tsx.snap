// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<DocumentFragment>
  .emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-column-gap: 5px;
  column-gap: 5px;
}

.emotion-1 {
  margin: unset;
  font-family: 'Euclid Circular A','Helvetica Neue',Helvetica,Arial,sans-serif;
  color: #001E2B;
  font-size: 32px;
  line-height: 40px;
  font-weight: 400;
  font-family: 'MongoDB Value Serif','Times New Roman',serif;
  color: #00684A;
  margin-top: 24px;
  margin-bottom: 8px;
  color: var(--heading-color-primary);
  margin-top: 16px;
  margin-bottom: 24px;
}

.emotion-2 {
  position: absolute;
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
  padding: 0 10px;
  visibility: hidden;
}

.emotion-3 {
  color: #889397;
  vertical-align: middle;
  margin-top: -2px;
}

.emotion-4 {
  display: inline;
  left: 0;
  top: 0;
  margin-top: -85px;
  position: absolute;
  padding-bottom: 2px;
}

@media only screen and (max-width: 767px) {
  .emotion-4 {
    margin-top: -85px;
  }
}

.emotion-5 {
  position: relative;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  min-width: 145px;
}

.emotion-5 [data-theme] {
  width: 310px;
}

@media only screen and (max-width: 1024px) {
  .emotion-5 {
    display: none;
  }
}

.emotion-6 {
  -webkit-appearance: none;
  -moz-appearance: none;
  -ms-appearance: none;
  appearance: none;
  padding: 0;
  margin: 0;
  background-color: transparent;
  border: 1px solid transparent;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
  -webkit-transition: all 150ms ease-in-out;
  transition: all 150ms ease-in-out;
  position: relative;
  -webkit-text-decoration: none;
  text-decoration: none;
  cursor: pointer;
  z-index: 0;
  font-family: 'Euclid Circular A','Helvetica Neue',Helvetica,Arial,sans-serif;
  border-radius: 6px;
  background-color: #F9FBFA;
  border-color: #889397;
  color: #001E2B;
  font-size: 13px;
  line-height: 20px;
  font-weight: 500;
  height: 36px;
  border-top-right-radius: 0px;
  border-bottom-right-radius: 0px;
  z-index: 1;
}

.emotion-6:focus-visible,
.emotion-6[data-focus="true"] {
  outline: none;
}

.emotion-6:active,
.emotion-6[data-active="true"],
.emotion-6:focus,
.emotion-6:hover {
  -webkit-text-decoration: none;
  text-decoration: none;
}

.emotion-6:focus-visible,
.emotion-6[data-focus="true"] {
  color: #001E2B;
}

.emotion-6:hover,
.emotion-6[data-hover="true"],
.emotion-6:active,
.emotion-6[data-active="true"] {
  color: #001E2B;
  background-color: #FFFFFF;
  box-shadow: 0 0 0 3px #E8EDEB;
}

.emotion-6:focus-visible,
.emotion-6[data-focus="true"] {
  background-color: #FFFFFF;
  box-shadow: 0 0 0 2px #FFFFFF,0 0 0 4px #0498EC;
}

.emotion-6,
.emotion-6:hover,
.emotion-6:focus,
.emotion-6:focus-visible,
.emotion-6:active {
  border-right-color: #889397;
}

.emotion-7 {
  overflow: hidden;
  position: absolute;
  top: 0;
  left: 0;
  right: 0;
  bottom: 0;
  border-radius: 5px;
}

.emotion-8 {
  display: grid;
  grid-auto-flow: column;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  height: 100%;
  width: 100%;
  position: relative;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  z-index: 0;
  -webkit-transition: all 150 ease-in-out;
  transition: all 150 ease-in-out;
  padding: 0 11px;
  gap: 6px;
}

.emotion-9 {
  color: #889397;
  height: 16px;
  width: 16px;
  justify-self: right;
}

.emotion-10 {
  -webkit-appearance: none;
  -moz-appearance: none;
  -ms-appearance: none;
  appearance: none;
  padding: 0;
  margin: 0;
  background-color: transparent;
  border: 1px solid transparent;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
  -webkit-transition: all 150ms ease-in-out;
  transition: all 150ms ease-in-out;
  position: relative;
  -webkit-text-decoration: none;
  text-decoration: none;
  cursor: pointer;
  z-index: 0;
  font-family: 'Euclid Circular A','Helvetica Neue',Helvetica,Arial,sans-serif;
  border-radius: 6px;
  background-color: #F9FBFA;
  border-color: #889397;
  color: #001E2B;
  font-size: 13px;
  line-height: 20px;
  font-weight: 500;
  height: 36px;
  border-top-left-radius: 0px;
  border-bottom-left-radius: 0px;
  margin-left: -1px;
  width: 32px;
}

.emotion-10:focus-visible,
.emotion-10[data-focus="true"] {
  outline: none;
}

.emotion-10:active,
.emotion-10[data-active="true"],
.emotion-10:focus,
.emotion-10:hover {
  -webkit-text-decoration: none;
  text-decoration: none;
}

.emotion-10:focus-visible,
.emotion-10[data-focus="true"] {
  color: #001E2B;
}

.emotion-10:hover,
.emotion-10[data-hover="true"],
.emotion-10:active,
.emotion-10[data-active="true"] {
  color: #001E2B;
  background-color: #FFFFFF;
  box-shadow: 0 0 0 3px #E8EDEB;
}

.emotion-10:focus-visible,
.emotion-10[data-focus="true"] {
  background-color: #FFFFFF;
  box-shadow: 0 0 0 2px #FFFFFF,0 0 0 4px #0498EC;
}

.emotion-10:focus,
.emotion-10:focus-visible,
.emotion-10:hover {
  z-index: 1;
}

.emotion-10,
.emotion-10:hover,
.emotion-10:focus,
.emotion-10:focus-visible,
.emotion-10:active {
  border-left-color: #889397;
}

.emotion-10 svg {
  color: #3D4F58;
}

.emotion-13 {
  color: #001E2B;
  height: 16px;
  width: 16px;
  justify-self: right;
}

.emotion-13 .lg-ui-button-0000:hover,
.emotion-13 .lg-ui-button-0000:active {
  color: currentColor;
}

.emotion-14 {
  display: none;
}

<section>
    <div
      class="emotion-0"
    >
      <h1
        class="contains-headerlink emotion-1"
        weight="medium"
      >
        Connect to your MongoDB instance
        <a
          class="headerlink emotion-2"
          href="#"
          title="Permalink to this heading"
        >
          <svg
            aria-label="Link Icon"
            class="emotion-3"
            fill="none"
            height="12"
            role="img"
            viewBox="0 0 16 16"
            width="12"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M6.039 9.953a3.596 3.596 0 0 1-.327-.38l1.45-1.445a1.553 1.553 0 0 0 2.496.423l2.885-2.87a1.57 1.57 0 0 0 .01-2.213 1.553 1.553 0 0 0-2.203-.01l-.379.377A.995.995 0 0 1 8.56 3.83a1.005 1.005 0 0 1 .006-1.418l.38-.377a3.542 3.542 0 0 1 5.024.023 3.58 3.58 0 0 1-.022 5.047l-2.884 2.871a3.542 3.542 0 0 1-5.025-.022Z"
              fill="currentColor"
            />
            <path
              d="M9.961 6.047c.12.12.228.248.327.38l-1.45 1.445a1.553 1.553 0 0 0-2.496-.423l-2.885 2.87a1.57 1.57 0 0 0-.01 2.213 1.553 1.553 0 0 0 2.203.01l.379-.377a.995.995 0 0 1 1.411.006 1.005 1.005 0 0 1-.006 1.418l-.38.377a3.542 3.542 0 0 1-5.024-.023 3.58 3.58 0 0 1 .022-5.047l2.884-2.871a3.542 3.542 0 0 1 5.025.022Z"
              fill="currentColor"
            />
          </svg>
          <div
            class="emotion-4"
            id=""
          />
        </a>
      </h1>
      <div
        class="emotion-5"
        data-lgid="lg-split_button"
        data-testid="lg-split_button"
      >
        <button
          aria-disabled="false"
          class="lg-ui-button-0000 emotion-6"
          data-lgid="lg-split_button-button"
          data-testid="lg-split_button-button"
          type="button"
        >
          <div
            class="emotion-7"
          />
          <div
            class="emotion-8"
          >
            <svg
              alt=""
              aria-hidden="true"
              class="emotion-9"
              fill="none"
              height="16"
              role="presentation"
              viewBox="0 0 16 16"
              width="16"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                clip-rule="evenodd"
                d="M1 5.714v4.572C1 11.233 1.768 12 2.714 12H5.75V7.11c0-.566.227-1.108.63-1.504l2.294-2.252c.1-.099.21-.186.326-.262v-.378C9 1.768 8.232 1 7.286 1H5.8v3.429c0 .71-.576 1.285-1.286 1.285H1Zm8-.928L7.257 6.498A.857.857 0 0 0 7 7.11v.688h3.01a.857.857 0 0 0 .857-.858V4h-.717a.857.857 0 0 0-.6.246l-.55.54ZM4.867 1H4.15a.857.857 0 0 0-.601.246L1.257 3.498A.857.857 0 0 0 1 4.11v.688h3.01a.857.857 0 0 0 .857-.858V1ZM7 12V8.714H10.514c.71 0 1.286-.575 1.286-1.285V4h1.486C14.233 4 15 4.768 15 5.714v7.572c0 .947-.768 1.714-1.714 1.714H8.714A1.714 1.714 0 0 1 7 13.286V12Z"
                fill="currentColor"
                fill-rule="evenodd"
              />
            </svg>
            Copy page
          </div>
        </button>
        <button
          aria-controls=""
          aria-disabled="false"
          aria-expanded="false"
          aria-haspopup="true"
          aria-label="More options"
          class="lg-ui-button-0000 emotion-10"
          data-lgid="lg-split_button-trigger"
          data-testid="lg-split_button-trigger"
          type="button"
        >
          <div
            class="emotion-7"
          />
          <div
            class="emotion-8"
          >
            <svg
              aria-label="Caret Down Icon"
              class="emotion-13"
              fill="none"
              height="16"
              role="img"
              viewBox="0 0 16 16"
              width="16"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M8.679 10.796a.554.554 0 0 1-.858 0L4.64 6.976C4.32 6.594 4.582 6 5.069 6h6.362c.487 0 .748.594.43.976l-3.182 3.82Z"
                fill="currentColor"
              />
            </svg>
          </div>
        </button>
        <span
          class="emotion-14"
        />
      </div>
    </div>
  </section>
</DocumentFragment>
`;
